.mcp.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Would it be possible to move this somewhere else than the root or does it have to be here? I feel like having a lot of files at the root pollutes the repo (same for entrypoint which can probably be moved to twenty-docker or utils?)